core/lexicon/country/tr.inc.php

```diff
@@ -247,11 +247,13 @@
 $_country_lang["tg"] = 'Togo'; // Togo
 $_country_lang["th"] = 'Tayland'; // Thailand
 $_country_lang["tj"] = 'Tacikistan'; // Tajikistan
-$_country_lang["tk"] = 'Tokelau'; // Tokelau
+$_country_lang["tk"] = 'Tokelau
+'; // Tokelau
```

Comment:
I don't think this is supposed to happen.

Reply:
This issue should be created inside of Crowdin and has to be changed there.

Reply:
Who's able to do that? @matdave maybe?

Reply:
The MODX Crowdin Managers are @opengeek, @jaygilmore, @theboxer, @sepiariver